demos/benchmark/assets/img_benchmark_cogwheel_argb.c

```diff
@@ -1,434 +1,136 @@
+#if defined(LV_LVGL_H_INCLUDE_SIMPLE)
+#include "lvgl.h"
+#elif defined(LV_BUILD_TEST)
 #include "../../../lvgl.h"
+#else
+#include "lvgl/lvgl.h"
+#endif
 
 #if LV_USE_DEMO_BENCHMARK
-
 #ifndef LV_ATTRIBUTE_MEM_ALIGN
-    #define LV_ATTRIBUTE_MEM_ALIGN
+#define LV_ATTRIBUTE_MEM_ALIGN
 #endif
 
-#ifndef LV_ATTRIBUTE_IMAGE_IMG_COGWHEEL_ARGB
-    #define LV_ATTRIBUTE_IMAGE_IMG_COGWHEEL_ARGB
+#ifndef LV_ATTRIBUTE_IMG_DUST
```

Comment:
Why is it called `DUST`?

Reply:
@anaGrad I see all the files have been changes with this define.
I suppose this is a python script effect, please have it fixed all over.
Thanks.

Reply:
Is there any convention on the naming of the images? 

- I notice there is an inconsistency in the names (for example, in the assets of widgets, some images have widget in the name, others don't). 
![image](https://github.com/lvgl/lvgl/assets/146175511/aadf69bf-e007-402f-8dcf-9c6411367cd5)

- The names of the .c files is different from that of the .png file (for example in benchmark/assets). 
![image](https://github.com/lvgl/lvgl/assets/146175511/aa89a92a-d256-4217-9d74-08b41097bc0b)

- There are multiple pngs with different names in different folders for what looks like the same image and I was wondering why is it like that (example: lvgl logo in demo/render vs demo/widgets) 
![image](https://github.com/lvgl/lvgl/assets/146175511/f86edba0-a810-4419-a02f-9b2848286e14)

Reply:
I don't understand neither. It's used in all old .c images and I simply copied the original headers as template to python script.

Reply:
The name of the image should be used in the ATTRIBUTE name. See [this](https://github.com/lvgl/lvgl/blob/release/v8.3/demos/widgets/assets/img_demo_widgets_avatar.c#L10) from v8.3.

The names are really not consistent now. We can work on that too in an other PR.

Reply:
Yes, that ATTRIBUTE has been fixed.

We will address separately the naming files .png vs .c It really has to be a consistency and avoid duplication.